### PR TITLE
Blog: Separate INFO and DEBUG logs

### PR DESCRIPTION
**Pull request by liangwen12year**
Signed-off-by: Wen Liang <liangwen12year@gmail.com>

### CODE REVIEW

File: blog/_posts/2020-10-08-separate-info-and-debug-logs.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+```
+
+The following Ansible logs is extracted from same qemu testing result after the bug fixed: 
+
+![](images/wenliang_blog_figure3.png)
+
+
+## Demo video
+
+I made a demo video to show the bugs and refactoring logging of network module after bug fixed,
```
Comment (richm) [inline blog/_posts/2020-10-08-separate-info-and-debug-logs.md:83]: Do you want to include a link to the video?

**Comment by richm** [comment]
Here is the PR to add a how to contribute a blog post: https://github.com/linux-system-roles/linux-system-roles.github.io/pull/40/files

File: blog/_posts/2020-10-08-separate-info-and-debug-logs.md (hunk 1 above)
Comment (liangwen12year) [reply blog/_posts/2020-10-08-separate-info-and-debug-logs.md:83]: I will, ideally, i want to include the video directly, thanks, Rich!

Comment (liangwen12year) [reply blog/_posts/2020-10-08-separate-info-and-debug-logs.md:83]: @richm  video added, thanks

**Review by richm — APPROVED** [review]
lgtm

**Pull request merged**